duckduckgo/Android
# update to use raster icon

Asana Issue URL: https://app.asana.com/0/392891325557410/531093953779309

## Description

Use raster icon to avoid vector icon corruption on some devices/configurations


## Steps to Test this PR:
1.  Launch the app on a device known to have corrupted search icon on home screen - icon should look fine
1.  Start an autocomplete - queries should also have the same search icon
